.env.example

```diff
@@ -0,0 +1,4 @@
+dbHost=ezxssdb
+dbUser=root
+dbPassword=changeme
```

Comment:
Does this still work with quotes around them? e.g. I have a password with special characters I need to add quotes around the password

Reply:
I will test and let you know

Reply:
Initially I was having trouble with this, however it seems to be a [docker-compose issue](https://github.com/docker/compose/issues/2854) that was present in the version of docker-compose I was using (1.25.0). It appears it was fixed in docker-compose version 1.26.0 and as such I have updated this file to use double quotes as well as added a note in the readme.